src/lib/components/chat-message.svelte

```diff
@@ -1,9 +1,18 @@
 <script lang="ts">
 	export let bubble = false
 	export let myMessage = false
+
+	const isFF = () => {
+		let browserInfo = navigator.userAgent
+		if (browserInfo.includes('Firefox')) {
+			return true
+		} else {
+			return false
+		}
```

Comment:
```suggestion
		return browserInfo.includes('Firefox'))
```

I would simplify it to this, because you can infer the type and value from the function name.

Reply:
oh yeah that's much smarter 😝
Thank you!!